README.md

```
@@ -267,6 +264,22 @@ Install Kr2FA using one of the following methods:
    /2fa <6-digit-code>
    ```
 
+###  Configuration
```

markdownlint-cli2 (0.18.1)

Multiple spaces after hash on atx style heading

(MD019, no-multiple-space-atx)